examples/abitest/module_0/rust/tests/integration_test.rs

```diff
@@ -45,16 +42,11 @@ fn build_wasm() -> anyhow::Result<HashMap<String, Vec<u8>>> {
     })
 }
 
-static LOG_INIT_ONCE: Once = Once::new();
-
 async fn setup() -> (
     Arc<oak_runtime::Runtime>,
     OakAbiTestServiceClient<tonic::transport::Channel>,
 ) {
-    LOG_INIT_ONCE.call_once(|| {
-        // Logger panics if it is initalized more than once.
-        env_logger::init();
-    });
+    let _ = env_logger::builder().is_test(true).try_init();
```

Comment:
Is it ok to ignore the result?

Reply:
I think so &ndash; using `try_init()` allows us to drop the `std::sync::Once` in favour of the `log` crate's [internal synchronization](https://docs.rs/log/0.4.11/src/log/lib.rs.html#1264-1285), but also means that a second attempt to initialize logging (e.g. in a second test) will return an error that needs to be ignored.